core/store/src/test/java/uk/gov/gchq/gaffer/store/schema/SchemaElementDefinitionValidatorTest.java

```diff
@@ -624,4 +624,17 @@ public void shouldValidateAndReturnFalseWhenAggregatorSelectionHasUnknownPropert
                 ),
                 result.getErrors());
     }
+
+    @Test
+    public void shouldValidateAndReturnTrueWhenEdgeDirectionUnset() {
```

Comment:
It's unfortunate we cannot easily ensure there is actually a warning logged.

Reply:
I have tested manually but yes, no easy way to test logging

Reply:
There are some ways to do it with Reload4j appenders and/or mocking. We wouldn't want to bring that in here though. It might be something for a future issue - code for testing that logging works could sit in a test util class.